roles/confluent.variables/defaults/main.yml

```diff
@@ -7,6 +7,10 @@ confluent_full_package_version: "{{ confluent_package_version + '-1' }}"
 confluent_package_redhat_suffix: "{{ '-' + confluent_full_package_version if confluent_full_package_version != '' else ''}}"
 confluent_package_debian_suffix: "{{ '=' + confluent_full_package_version if confluent_full_package_version != '' else ''}}"
 
+### Boolean to mask secrets in playbook output
+mask_secrets: true
+mask_diff: true
```

Comment:
need an annotation for the variable, also to run `python docs.py` will write the docs for this var

Reply:
Done